test/e2e/performanceprofile/functests/utils/consts.go

```diff
@@ -44,8 +45,14 @@ func init() {
 
 	NodesSelector = os.Getenv("NODES_SELECTOR")
 
-	NodeSelectorLabels = map[string]string{
-		fmt.Sprintf("%s/%s", LabelRole, RoleWorkerCNF): "",
+	if !hypershift.IsHypershiftCluster() {
+		NodeSelectorLabels = map[string]string{
+			fmt.Sprintf("%s/%s", LabelRole, RoleWorkerCNF): "",
+		}
+	} else {
+		NodeSelectorLabels = map[string]string{
+			fmt.Sprintf("%s/%s", LabelRole, RoleWorker): "",
+		}
```

Comment:
this can be simplified like
```
roleValue := RoleWorkerCNF
if hypershift.IsHyperShiftCluster() {
    roleValue = RoleWorker
}
NodeSelectorLabels = map[string]string{
		fmt.Sprintf("%s/%s", LabelRole, RoleValue): "",
}
```

but not worth resubmitting for this